mesonbuild/mconf.py

```diff
@@ -152,16 +185,16 @@ def run(options):
     try:
         c = Conf(builddir)
         save = False
```

Comment:
I think it makes more sense to nest this whole block instead:
```python
if not c.default_values_only:
    if len(options.cmd_line_options) > 0:
        ....
     elif options.clearcache:
         ....
```

Reply:
Did something similar without an extra level of indentation.